netweaver/extract_installer.sls

```diff
@@ -0,0 +1,22 @@
+{%- from "netweaver/map.jinja" import netweaver with context -%}
+{% set host = grains['host'] %}
+{% for node in netweaver.nodes if node.host == host and (netweaver.sapcar_exe_file is defined and netweaver.swpm_sar_file and netweaver.swpm_extract_dir is defined) %}
+{% set instance = '{:0>2}'.format(node.instance) %}
+{% set name = '{}_{}'.format(node.sid, instance) %}
+
```

Comment:
Why so many new lines?

Reply:
Thanks, there is extra line from previous removed line. I corrected this.